pages/test_analytics/importing_junit_xml.md

```diff
@@ -10,7 +10,7 @@ The following attributes are mandatory for the `<testcase>` element:
 * `classname`: full class name for the class the test method is in.
 * `name`: name of the test method.
 
-To learn more about the JUnit XML file format, see [JUnit XML reporting file format](https://llg.cubic.org/docs/junit/) from Linux Lighting Group.
+To learn more about the JUnit XML file format, see [JUnit XML reporting file format](https://github.com/testmoapp/junitxml#basic-junit-xml-structure).
```

Comment:
Thanks @mbelton-buildkite , looks great!

Do you think it should link to https://github.com/testmoapp/junitxml (top of the README) instead rather than the 'Basic JUnit XML structure' heading (https://github.com/testmoapp/junitxml#basic-junit-xml-structure) on this page?

Happy with either choice.

Reply:
Sure. I'll update the link text to match the new destination, too.

```suggestion
To learn more about the JUnit XML file format, see [Common JUnit XML format & examples
](https://github.com/testmoapp/junitxml).
```

Reply:
Good pick-up, I missed that!